tiledb/ml/models/sklearn.py

```diff
@@ -5,7 +5,6 @@
 import numpy as np
 import json
 import tiledb
-import tiledb.cloud
```

Comment:
:warning:  There is still a reference to `tiledb.cloud.array.update_file_properties`

Reply:
@gsakkis nice catch. The same stands for pytorch and tensorflow model classes.

Reply:
So do you think that would be a good idea to import `tiledb.cloud` before this statement?

Reply:
Import aside, the whole `tiledb.cloud.array.update_file_properties` call should be extracted as a function to cloud_utils; only the name of the framework changes:
```py
tiledb/ml/models/pytorch.py
127:            tiledb.cloud.array.update_file_properties(
128-                uri=self.uri,
129-                file_type=FILETYPE_ML_MODEL,
130-                file_properties={
131-                    FilePropertyName_ML_FRAMEWORK: "PYTORCH",
132-                    FilePropertyName_STAGE: "STAGING",
133-                    FilePropertyName_PYTHON_VERSION: platform.python_version(),
134-                    FilePropertyName_ML_FRAMEWORK_VERSION: torch.__version__,
135-                },
136-            )

tiledb/ml/models/sklearn.py
87:            tiledb.cloud.array.update_file_properties(
88-                uri=self.uri,
89-                file_type=FILETYPE_ML_MODEL,
90-                file_properties={
91-                    FilePropertyName_ML_FRAMEWORK: "SKLEARN",
92-                    FilePropertyName_STAGE: "STAGING",
93-                    FilePropertyName_PYTHON_VERSION: platform.python_version(),
94-                    FilePropertyName_ML_FRAMEWORK_VERSION: sklearn.__version__,
95-                },
96-            )

tiledb/ml/models/tensorflow.py
184:            tiledb.cloud.array.update_file_properties(
185-                uri=self.uri,
186-                file_type=FILETYPE_ML_MODEL,
187-                file_properties={
188-                    FilePropertyName_ML_FRAMEWORK: "TENSORFLOW",
189-                    FilePropertyName_STAGE: "STAGING",
190-                    FilePropertyName_PYTHON_VERSION: platform.python_version(),
191-                    FilePropertyName_ML_FRAMEWORK_VERSION: tf.__version__,
192-                },
193-            )
```

Reply:
That might fix the issue with import as well.

Reply:
I was about to include this in what I'm working at the moment but we can do it here.

Reply:
I did the changes here no worries.